python_ta/checkers/condition_logic_checker.py

```diff
@@ -0,0 +1,102 @@
+"""
+Check for redundant If or While conditions in functions based on z3 constraints
+"""
+
+from typing import Any, Union
+
+try:
+    import z3
+
+    from ..cfg.graph import Z3Environment
+
+    z3_dependency_available = True
+except ImportError:
+    z3 = Any
+    Z3Environment = Any
+    z3_dependency_available = False
+
+from astroid import nodes
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import only_required_for_messages
+from pylint.lint import PyLinter
+
+
+class ConditionLogicChecker(BaseChecker):
+    name = "redundant-condition"
+    msgs = {
+        "R9900": (
+            """This condition will always evaluate to True;
+             consider removing redundant check.""",
```

Comment:
Remove this second part and end the sentence after `True`. ("consider" does not convey the right level of instruction here, as we really do want students to remove the redundant condition)